src/arrayMethodSort.js

```diff
@@ -3,9 +3,33 @@
 /**
  * Implement method Sort
  */
+const compareStandart = function (a, b) {
+  if (String(a) > String(b)) {
+    return 1;
+  }
+
+  if (String(a) < String(b)) {
+    return -1;
+  } else {
+    return 0;
+  }
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = compareStandart) {
+    let temporaryItem;
+
+    for (let j = 0; j < this.length; j++) {
+      for (let i = 0; i < this.length; i++) {
+        if (compareFunction(this[i], this[i + 1]) > 0) {
+          temporaryItem = this[i];
+          this[i] = this[i + 1];
+          this[i + 1] = temporaryItem;
+        }
+      }
```

Comment:
The current implementation of the bubble sort algorithm will perform unnecessary comparisons because it does not account for the fact that after each pass, the largest element is already sorted at the end of the array. You can optimize the inner loop to run only until `this.length - j - 1` to avoid these redundant comparisons.